vms/volunteer/tests/test_volunteerProfile.py

```diff
@@ -302,7 +299,7 @@ def test_corrupt_resume_uploaded(self):
         try:
             PyPDF2.PdfFileReader(open(path, 'rb'))
         except PdfReadError:
-            print('Some error while upload/download')
+            self.fail('Some error while upload/download')
```

Comment:
Message is still same :confused: , please change it to something similar to `Uploaded resume is corrupted`